<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProductBasketInterfaceItem</class>
 <widget class="QFrame" name="ProductBasketInterfaceItem">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>94</width>
    <height>123</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>30</width>
    <height>123</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>200</width>
    <height>123</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QFrame	{
	background-color: transparent;
	border: 0px;
}</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::StyledPanel</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Raised</enum>
  </property>
  <widget class="VR::VRQFrame" name="m_pVRQFrameProductItem">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>15</y>
     <width>81</width>
     <height>103</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>81</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">QFrame	{
	border: 1px solid rgb(29,29,29);
	border-radius: 0px;
	background-color: rgb(67,67,67);
}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="m_pLabelProductInfo">
    <property name="geometry">
     <rect>
      <x>1</x>
      <y>77</y>
      <width>79</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>7</pointsize>
      <weight>75</weight>
      <bold>true</bold>
      <strikeout>false</strikeout>
      <stylestrategy>PreferDefault</stylestrategy>
     </font>
    </property>
    <property name="acceptDrops">
     <bool>true</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
	color: rgb(204,204,204);
	background-color:rgb(102,102,102);
	border-radius:0px;
	border:0px;
}

/*
	border: 1px solid rgb(67,67,67);
*/</string>
    </property>
    <property name="text">
     <string>Samsung Galaxy Tab (R) 3 8.0 (Wi-Fi) White</string>
    </property>
    <property name="textFormat">
     <enum>Qt::PlainText</enum>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
    <property name="margin">
     <number>-1</number>
    </property>
    <property name="indent">
     <number>0</number>
    </property>
   </widget>
   <widget class="QLabel" name="m_pLabelProductImage">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>81</width>
      <height>79</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel	{
	border: 1px solid rgb(29,29,29);
	border-radius: 0px;
	padding:0px;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="textFormat">
     <enum>Qt::PlainText</enum>
    </property>
    <property name="pixmap">
     <pixmap>../../../../../../../../../Matej/Images/blueFlowers.bmp</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="margin">
     <number>0</number>
    </property>
    <property name="indent">
     <number>0</number>
    </property>
   </widget>
   <widget class="QFrame" name="m_pFrameProductItemHover">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>1</x>
      <y>1</y>
      <width>79</width>
      <height>77</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>81</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QFrame	{
	border:0px;
	border-radius: 0px;
	background-color: rgba(67,67,67,200);
}

/*
	border: 1px solid rgb(29,29,29);
*/</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="m_pPushButtonDetails">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>10</y>
       <width>79</width>
       <height>18</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>7</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">/*
QPushButton {
	border: 2px solid #8f8f91;
	border-radius: 6px;
	background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                       stop: 0 #f6f7fa, stop: 1 #dadbde);
	min-width: 75px;
	max-width: 75px;
}
	background-color: rgb(204,204,204);

*/
QPushButton {
	color: rgb(204,204,204);
	border: 1px solid rgb(67,67,67);
	border-radius: 6px;
	background-color:rgb(24,24,24);

	width: 81px;
}

QPushButton:hover {
	color: white;
}



</string>
     </property>
     <property name="text">
      <string>Details</string>
     </property>
    </widget>
    <widget class="QPushButton" name="m_pPushButtonRemove">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>30</y>
       <width>79</width>
       <height>18</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>7</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
	color: rgb(204,204,204);
	border: 1px solid rgb(67,67,67);
	border-radius: 6px;
	background-color:rgb(24,24,24);

	width: 81px;
}

QPushButton:hover {
	color: white;
}</string>
     </property>
     <property name="text">
      <string>Remove</string>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="m_pDoubleSpinBoxQuantity">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>50</y>
       <width>79</width>
       <height>18</height>
      </rect>
     </property>
     <property name="maximumSize">
      <size>
       <width>79</width>
       <height>18</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>7</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">QDoubleSpinBox {
	width: 81px;
	border-radius: 6px;
	padding-left: 5px;
	padding-right: 5px;

	color: rgb(204,204,204);
	border: 1px solid rgb(67,67,67);
	background-color:rgb(24,24,24);
}	

QDoubleSpinBox::up-button {
	subcontrol-origin: border;
	subcontrol-position: right;
	border-width: 0px;
}

QDoubleSpinBox::up-arrow {
	image: url(:/arrow_right.png);
	width: 10px;
	height: 10px;
}

QDoubleSpinBox::down-button {
	subcontrol-origin: border;
	subcontrol-position: left;
	border-width: 0px;
}

QDoubleSpinBox::down-arrow {
	image: url(:/arrow_left.png);
	width: 10px;
	height: 10px;
}</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QLabel" name="m_pLabelBasketProductPrice">
   <property name="geometry">
    <rect>
     <x>21</x>
     <y>5</y>
     <width>47</width>
     <height>15</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>7</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="acceptDrops">
    <bool>true</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	color: rgb(204,204,204);
	border-radius: 5px;
	border: 1px solid rgb(18,18,18);
	background-color:rgb(24,24,24);
}

/*
rgb(70,70,70);*/</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>VR::VRQFrame</class>
   <extends>QFrame</extends>
   <header location="global">vrqframe.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
